jwql/tests/test_api_views.py

```diff
@@ -90,10 +99,8 @@ def test_api_views(url):
 
     try:
         url = request.urlopen(url)
-    except error.HTTPError as e:
-        if e.code == 502:
-            pytest.skip("Server problem")
-        raise(e)
+    except (error.HTTPError, error.URLError):
```

Comment:
I included the URL catch in case some of the specific data referenced on central storage goes missing again.  I can let that fail instead if you prefer.